ui/src/app/utils/access-utils.tsx

```diff
@@ -30,6 +30,13 @@ import {
   MILLIS_PER_DAY,
 } from './dates';
 
+export enum AccessModulesStatus {
```

Comment:
Not sure but we may want to qualify this with something like "expiration status" or "renewal status" because this is specifically viewing a module's status with that perspective